### DISCUSSION TITLE
Long press to download feed enclosures (eg MP3s)

### DISCUSSION THREAD

**Discussion by doriantaylor** [spacecowboy/Feeder#314] [Ideas]
It would be really handy if the long-press context menu had a `Download` item for media links. Happy to write the code myself if somebody can point me in the general direction.

**Comment by spacecowboy** [comment]
https://github.com/spacecowboy/Feeder/blob/e89aa72b707012f50bdfc1f4e134b490fffaf272/app/src/main/java/com/nononsenseapps/feeder/ui/compose/feedarticle/ReaderView.kt#L269

Add a long click action somewhere here.
